src/components/Admin/UserPage/ManageUsers.tsx

```diff
@@ -103,6 +198,7 @@ const ManageUsers: () => JSX.Element = () => {
                 <Tr>
                   <Th>{t('users.name')}</Th>
                   <Th>{t('email')}</Th>
+                  <Th>{t('users.removeAdmin.button')}</Th>
```

Comment:
Line 194 (7 lines above this), should the translation variable be `users.adminHeading` instead (just change the variable name to acknowledge its the heading for admin users, not all users). There's also a typo in that the text is "Current Admins Users" instead of "Current Admin Users".

Reply:
I just updated the headings with the new change just showing admin users, so lmk if my update makes more sense now